compiler/include/flags_list.h

```diff
@@ -185,6 +185,7 @@ symbolFlag( FLAG_INSERT_LINE_FILE_INFO , ypr, "insert line file info" , ncm )
 symbolFlag( FLAG_INSTANTIATED_PARAM , npr, "instantiated param" , "this formal is bound to a param" )
 symbolFlag( FLAG_INSTANTIATED_GENERIC , npr, "instantiated generic" , "this is an instantiation of a generic" )
 symbolFlag( FLAG_INSTANTIATED_FROM_ANY , npr, "instantiated from any" , "this is an instantiation from any type" )
+symbolFlag( FLAG_INTENT_REF_MAYBE_CONST_FORMAL, ypr, "intent ref maybe const formal", "The intent for this formal is ref if it is modified, const otherwise" )
```

Comment:
Re the comment - it's `ref` or `const ref` - "The intent for this formal is ref or const ref depending on whether or not it is modified"

Reply:
I updated the comment to say `const ref` instead of just `const`.